hydra-node/src/Hydra/Chain/Direct.hs

```diff
@@ -158,6 +159,7 @@ withDirectChain tracer networkId iocp socketPath keyPair party cardanoKeys callb
         SomeOnChainHeadState $
           idleOnChainHeadState
             networkId
+            (cardanoKeys \\ [verificationKey wallet])
```

Comment:
suggestion: Perhaps worthwhile to detail the comment on line 151 so that this line makes more within the context?